### PR TITLE
Breaking Change: Automatic base64 encoding/decoding for secrets

### LINKED ISSUE #95: Add better handling/wrapping for Secrets
Currently the Secret `data` property is carried as Base64 encoded (unlike the GO SDK that does this for you), Pepr should have a wrapper for these functions, also should include an update to `HelloPepr` showing this:

Read from:
```ts
 if (data && data[p]) {
    return Buffer.from(data[p], "base64").toString("utf-8");
  }
```

Write back:
`Buffer.from(secret).toString("base64");`

### PR DESCRIPTION
**Pull request by jeff-mccoy**
This PR closes #95 by transparently handling Base64 encoding for the `data` field of Secrets. This behavior mirrors the Go K8s client behavior and eliminates the need for users to understand/worry about dealing with the encoding. 

Other options explored included extending new classes for ConfigMaps and Secrets, and using getters/setters, but these failed due to the nature of JS getter/setter support for modifying object properties.

### CODE REVIEW

File: hack/e2e.test.mjs [hunk 1]
```diff
@@ -247,6 +254,16 @@ async function waitForConfigMap(namespace, name) {
   }
 }
 
+async function waitForSecret(namespace, name) {
```
Comment (bdw617) [inline hack/e2e.test.mjs:257]: this looks like could be an endless loop?

**Review by bdw617 — COMMENTED** [review]
see comment

File: hack/e2e.test.mjs (hunk 1 above)
Comment (jeff-mccoy) [reply hack/e2e.test.mjs:257]: Yes it is, but the ava test config holds the timeout and will kill it.

Comment (bdw617) [reply hack/e2e.test.mjs:257]: ok

**Review by bdw617 — APPROVED** [review]
ok, if the ava stuff will kill it, prob good to document that though

**Pull request merged**